nexus/client.go

```diff
@@ -268,6 +291,8 @@ type ExecuteOperationOptions struct {
 	// Request ID that may be used by the server handler to dedupe this start request.
 	// By default a v4 UUID will be generated by the client.
 	RequestID string
+	// Links contain information about the caller event that made the Nexus call.
```

Comment:
Copy my suggestion for the docstring from `StartOperationOptions`.

Reply:
You resolved without changing.

Reply:
It's changed.